src/components/header/header-actions.astro

```diff
@@ -21,9 +21,6 @@ const IS_LIVE = false;
           </abbr>
           <span class="hidden md:inline">Code of Conduct</span>
         </ButtonLink>
-        <ButtonLink href="/programme/cfp" className="hidden md:block">
-          <span>Send your proposal!</span>
-        </ButtonLink>
```

Comment:
We will never find these lines again. So if someone else could please do a suggestion with commenting this out instead?

Reply:
GitHub doesn't allow it :D @rodrigogiraoserrao can you comment this part out instead?

Reply:
See #994

Reply:
Well, there are two other buttons around the one I deleted, so I didn't think this was going to be a problem :D Added it back.

Reply:
It probably would not have been a problem. You are correct. But I do know that I failed on that task once already.